lib/index.js

```diff
@@ -31,9 +33,20 @@ bedrock.events.on('bedrock.configure', function applyConfig() {
     throw new Error('"bedrock-config-yaml" must be the last import.');
   }
 
-  _applyConfig({configType: 'app'});
+  if(process.env.BEDROCK_CONFIG) {
+    applyConfigFromEnv();
+  } else {
+    _applyConfig({configType: 'app'});
+  }
 });
 
+export function applyConfigFromEnv() {
```

Comment:
We should prefix with underscore to denote that this is not part of a public API and is a test fixture.
```suggestion
export function _applyConfigFromEnv() {
```

Reply:
yeah, had that originally. I think I need to comment it's exported for test purposes only in that case